apisix/plugins/prometheus/exporter.lua

```diff
@@ -45,10 +45,10 @@ local DEFAULT_BUCKETS = { 1, 2, 5, 7, 10, 15, 20, 25, 30, 40, 50, 60, 70,
 local metrics = {}
 
 
-    local inner_tab_arr = {}
-local function gen_arr(...)
-    clear_tab(inner_tab_arr)
 
+local function gen_arr(...)
```

Comment:
We don't need this if we can't share the table.

Reply:
got it.